pkg/sql/schema_changer.go

```diff
@@ -988,9 +989,8 @@ func (s *SchemaChangeManager) Start(stopper *stop.Stopper) {
 		//
 		// Vivek wrote: I believe the only reason to pace these is to allow
 		// different schema changes to execute on different nodes and distribute
-		// the load. But that was a thought a long time ago. You can safely move
-		// the break inside the if statement.
-		delay := 360 * time.Second
+		// the load. But that was a thought a long time ago.
+		delay := time.Duration((30 + rand.Float64()*5) * float64(time.Second))
```

Comment:
Can you hoist this into constants at the top of the file?

![hoist](http://rmhoist.com/files/public/userfiles/1002/Spacemaster%20SX%20Aux%20Hoist%20RGB.jpg)

Reply:
I just read the comments on the issue this is referencing and there is a situation I'm concerned with. If a user dos the following:

```sql
BEGIN
ALTER TABLE bar ADD INDEX ... # Initiate the first schema change
ALTER TABLE foo ADD INDEX ... # Initiate the first schema change
COMMIT
```

We have the following happening:
1) The schema change adding the index on `bar` start.

   1) If this takes more than 6 minutes will the schema change manager
   begin to run the schema change for `foo`.
2) The schema change for `bar` finishes.
3) We begin to execute the schema changer for `foo` which has already
   started. What happens in this case?

Mind my gap of understanding here, but I don't know what will happen.
Reducing the timeout just makes this happen sooner. Does the session
block waiting to acquire the tablee schema change lease which it only
gets immediately after it finishes?

Reply:
Right, so there is an exponential backoff with a maximum of 1s here. I think once a manager begins executing the schema change there is no guarantee the session will block until all the changes are complete.